hibernate-core/src/main/antlr/org/hibernate/grammars/hql/HqlLexer.g4

```diff
@@ -93,6 +93,10 @@ UNICODE_ESCAPE
 	: 'u' HEX_DIGIT HEX_DIGIT HEX_DIGIT HEX_DIGIT
 	;
 
+BIN_LITERAL
```

Comment:
Can we use `BINARY_LITERAL`?  It's 3 extra characters and so much more immediately understandable.

Reply:
Sure.